### PR TITLE
added the new incident_id to the response object

### PR DESCRIPTION
**Pull request by kcjerrell**
# Description

[Explanation video](https://www.loom.com/share/99edc96597594079b6145067d8800db2)

This change is to the POST incident endpoint. All it does it add the new incident_id to the response object, so the front end can update local state without having to fetch all the incidents again.

## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?
- [x] npm run tests

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: api/allIncidents/adminIncidentsRouter.js [hunk 1]
```diff
@@ -173,8 +173,11 @@ router.post(
   validateAndSanitizeIncidentObject,
   async (req, res, next) => {
     Incidents.createIncident(req.sanitizedIncident)
-      .then(() => {
-        res.status(201).json({ message: 'Incident Successfully Created' });
+      .then((result) => {
+        res.status(201).json({
+          message: 'Incident Successfully Created',
+          incident_id: result.newIncidentId,
+        });
```
Comment (pflee1989) [inline api/allIncidents/adminIncidentsRouter.js:180]: Let's do it

**Review by pflee1989 — APPROVED** [review]

File: api/allIncidents/adminIncidentsRouter.js [hunk 2]
```diff
@@ -173,8 +173,11 @@ router.post(
   validateAndSanitizeIncidentObject,
   async (req, res, next) => {
     Incidents.createIncident(req.sanitizedIncident)
-      .then(() => {
-        res.status(201).json({ message: 'Incident Successfully Created' });
+      .then((result) => {
+        res.status(201).json({
```
Comment (pflee1989) [inline api/allIncidents/adminIncidentsRouter.js:177]: Let's go ahead and review this one with Paul.

**Review by pflee1989 — APPROVED** [review]

File: api/allIncidents/adminIncidentsRouter.js (hunk 2 above)
Comment (ryan-hamblin) [reply api/allIncidents/adminIncidentsRouter.js:177]: I think this is a simple and necessary change that will dramatically improve the user experience.

File: api/allIncidents/adminIncidentsRouter.js (hunk 1 above)
Comment (ryan-hamblin) [reply api/allIncidents/adminIncidentsRouter.js:180]: I'm resolving to help paul see that I looked over the code.

**Review by ryan-hamblin — APPROVED** [review]

**Pull request merged**